From 3168b1dedf88c9026683ed43ea337814003d6802 Mon Sep 17 00:00:00 2001
From: John Tsiombikas <nuclear@member.fsf.org>
Date: Sat, 1 Feb 2020 22:08:43 +0200
Subject: [PATCH] add a USE_X11 ifdef around the call to drop_xinput in
 src/dev.c to fix non-X11 build. This should fix github issue #19.

---
 src/dev.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/dev.c b/src/dev.c
index 8b72932..2074895 100644
--- a/src/dev.c
+++ b/src/dev.c
@@ -88,7 +88,9 @@ int init_devices(void)
 		return -1;
 	}
 
+#ifdef USE_X11
 	drop_xinput();
+#endif
 	return 0;
 }
 
